src/cswinrt/strings/additions/Microsoft.UI.Xaml.Media.Animation/Microsoft.UI.Xaml.Media.Animation.RepeatBehavior.cs

```diff
@@ -0,0 +1,186 @@
+
+namespace Microsoft.UI.Xaml.Media.Animation
+{
+    using global::Windows.Foundation;
+
+    [WindowsRuntimeMetadata("Microsoft.UI")]
+    [WindowsRuntimeClassName("Windows.Foundation.IReference<Microsoft.UI.Xaml.Media.Animation.RepeatBehavior>")]
+    [ABI.Microsoft.UI.Xaml.Media.Animation.RepeatBehaviorComWrappersMarshaller]
+    [StructLayout(LayoutKind.Sequential)]
+    public struct RepeatBehavior : IFormattable, IEquatable<RepeatBehavior>
+    {
+        internal static bool IsFinite(double value)
+        {
+            return !(double.IsNaN(value) || double.IsInfinity(value));
+        }
+
+        public RepeatBehavior(double count)
+        {
+            if (!IsFinite(count) || count < 0.0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(count));
+            }
+
+            Duration = new TimeSpan(0);
+            Count = count;
+            Type = RepeatBehaviorType.Count;
+        }
+
+        public RepeatBehavior(TimeSpan duration)
+        {
+            if (duration < new TimeSpan(0))
+            {
+                throw new ArgumentOutOfRangeException(nameof(duration));
+            }
+
+            Duration = duration;
+            Count = 0.0;
+            Type = RepeatBehaviorType.Duration;
+        }
+
+        public static RepeatBehavior Forever
+        {
+            get
+            {
+                RepeatBehavior forever = default;
+                forever.Type = RepeatBehaviorType.Forever;
+
+                return forever;
+            }
+        }
+
+        public bool HasCount
+        {
+            get
+            {
+                return Type == RepeatBehaviorType.Count;
+            }
+        }
+
+        public bool HasDuration
+        {
+            get
+            {
+                return Type == RepeatBehaviorType.Duration;
+            }
+        }
+
+        public double Count
+        {
+            readonly get; set;
+        }
+
+        public TimeSpan Duration
+        {
+            readonly get; set;
+        }
+
+        public RepeatBehaviorType Type
+        {
+            readonly get; set;
+        }
+
+        public override string ToString()
+        {
+            return InternalToString(null, null);
+        }
+
+        public string ToString(IFormatProvider formatProvider)
+        {
+            return InternalToString(null, formatProvider);
+        }
+
+        string IFormattable.ToString(string format, IFormatProvider formatProvider)
+        {
+            return InternalToString(format, formatProvider);
+        }
+
+        internal string InternalToString(string format, IFormatProvider formatProvider)
+        {
+            switch (Type)
+            {
+                case RepeatBehaviorType.Forever:
+
+                    return "Forever";
+
+                case RepeatBehaviorType.Count:
+
+                    global::System.Text.StringBuilder sb = new global::System.Text.StringBuilder();
```

Comment:
Also we should set `SkipLocalsInit` for all projection .dlls if we don't already. At least the merged .dll, as we own it.